scripts/verify-tsconfig.mjs

```diff
@@ -0,0 +1,28 @@
+import { promises as fs } from 'fs';
+import pathUtils from 'path';
+import { fileURLToPath } from 'url';
+
+const cwd = pathUtils.dirname(fileURLToPath(import.meta.url))
+
+// These are the packages we expect to _not_ be referenced in the root tsconfig.
+const IGNORE_LIST = new Set(['examples', 'types']);
+
+// Get reference paths from root tsconfig.json
+const rootTsconfig = JSON.parse(await fs.readFile('./tsconfig.json', { encoding: 'utf8'}));
```

Comment:
Tsconfig files can have comments in them and be valid - JSON.parse on the other hand will fail

Reply:
Ah, well we can cross that bridge when we get there.

I couldn't get JSON module imports to work on Node 14, is the only reason I'm doing it this way.